<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="3" skipped="0" tests="3" time="0.476" timestamp="2022-12-14T23:29:24.238021" hostname="fa2a076f34cb"><testcase classname="ros2_aruco.test.test_copyright" name="test_copyright" time="0.061"><failure message="AssertionError: Found errors&#10;assert 1 == 0">test/test_copyright.py:23: in test_copyright
    assert rc == 0, 'Found errors'
E   AssertionError: Found errors
E   assert 1 == 0</failure></testcase><testcase classname="ros2_aruco.test.test_flake8" name="test_flake8" time="0.232"><failure message="AssertionError: Found errors&#10;assert 1 == 0">test/test_flake8.py:23: in test_flake8
    assert rc == 0, 'Found errors'
E   AssertionError: Found errors
E   assert 1 == 0</failure></testcase><testcase classname="ros2_aruco.test.test_pep257" name="test_pep257" time="0.091"><failure message="AssertionError: Found code style errors / warnings&#10;assert 1 == 0">test/test_pep257.py:23: in test_pep257
    assert rc == 0, 'Found code style errors / warnings'
E   AssertionError: Found code style errors / warnings
E   assert 1 == 0</failure></testcase></testsuite></testsuites>